coala_langserver/langserver.py

```diff
@@ -3,76 +3,67 @@
 import socketserver
 import traceback
 
-from .jsonrpc import JSONRPC2Connection, ReadWriter, TCPReadWriter
+from pyls.jsonrpc.endpoint import Endpoint
+from pyls.jsonrpc.dispatchers import MethodDispatcher
+from pyls.jsonrpc.streams import JsonRpcStreamReader
+from pyls.jsonrpc.streams import JsonRpcStreamWriter
 from .log import log
 from .coalashim import run_coala_with_specific_file
 from .uri import path_from_uri
 from .diagnostic import output_to_diagnostics
 
 
-class ThreadingTCPServer(socketserver.ThreadingMixIn, socketserver.TCPServer):
-    pass
+class _StreamHandlerWrapper(socketserver.StreamRequestHandler, object):
+    '""A wrapper class that is used to construct a custom handler class.""'
```

Comment:
`'""` is not a docstring.
It is invalid, and you should not be submitted invalid syntax.
I already raised https://github.com/coala/coala-vs-code/issues/56 about this.
Find a solution.

Reply:
As you mentioned https://github.com/coala/coala-vs-code/issues/56 is now blocking this PR.